src/components/Spinner/Spinner.driver.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+
+export interface SpinnerDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
+}
+
+export const spinnerDriverFactory = (base: UniDriver): SpinnerDriver => {
+  return {
+    ...baseUniDriverFactory(base),
+    isMobile: async () => (await base.attr('data-mobile')) === 'true',
```

Comment:
The `isMobile` prop isn't needed for this component.
So you can just remove it

Reply:
Removed mobile from all cases inside component